DigitalLearningSolutions.Web/ViewComponents/DateInputViewComponent.cs

```diff
@@ -35,6 +34,13 @@ string hintText
             var dayValue = dayProperty?.GetValue(model)?.ToString();
             var monthValue = monthProperty?.GetValue(model)?.ToString();
             var yearValue = yearProperty?.GetValue(model)?.ToString();
+            var dayErrors = ViewData.ModelState[dayProperty?.Name]?.Errors;
+            var monthErrors = ViewData.ModelState[monthProperty?.Name]?.Errors;
+            var yearErrors = ViewData.ModelState[yearProperty?.Name]?.Errors;
+
+            var errorMessage = dayErrors?.Count > 0 ? dayErrors[0].ErrorMessage :
+                               monthErrors?.Count > 0 ? monthErrors[0].ErrorMessage :
+                               yearErrors?.Count > 0 ? yearErrors[0].ErrorMessage : null;

```

Comment:
Just trying to think if we'd ever want to display more than one error, but I don't think we would. :)